test/unit/server.test.ts

```diff
@@ -208,8 +208,8 @@ describe('server client', function () {
         context = client.__store.getStore().context;
```

Comment:
Since this is going to be called only once, you could make the assertions from L218-L219 and call `done()` here, right?

Reply:
True, but I didn't, because I wanted to make sure the error handler was called. Putting the assertions into the system under test means that the code could stop calling the error handler, and we wouldn't notice.

Reply:
This is inside the `errorHandler` right? So the assertions (+ the call to `done()`) would only be called if the `errorHandler` is called.

Reply:
Yes, but if it isn't, then you'd have a test timeout exception rather than a failing assertion.

Reply:
Ah ok, good point.